dependency_confusion.rego

```diff
@@ -1,12 +1,14 @@
-package policy
+# METADATA
+# title: Dependency Confusion
+# description: |
+#    Blocks dependency confusion
+package policy.v1
 
 import rego.v1
 
-# Returns a violation if the package appears to be a dependency confusion
 # METADATA
-# scope: rule
-# schemas:
-#   - data.issue: schema.issue
-issue contains "Package appears to be a dependency confusion" if {
-	data.issue.tag == "HM0018"
+# title: Dependency confusion
```

Comment:
For these ones where the rule title and the package title are the same or very similar, we could probably just leave out the rule title.

Reply:
Understood, will just leave it for now.